.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,76 @@
+name: alwaysai/codecs
```

Comment:
This should be `alwaysai/config-nodejs`, right?

Reply:
@ericvanbuhler Yes! Thank you! Copy and paste error